cg/meta/workflow/nf_analysis.py

```diff
@@ -308,26 +309,70 @@ def run_analysis(
             )
 
     @staticmethod
```

Comment:
Should it really be a staticmethod if you pass self?

Reply:
No, I removed it now.